Source/LibationWinForms/ProcessQueue/VirtualFlowControl.cs

```diff
@@ -190,7 +190,10 @@ private void AdjustScrollBar()
 				vScrollBar1.LargeChange = LargeScrollChange;
 
 				//https://stackoverflow.com/a/2882878/3335599
-				vScrollBar1.Maximum = VirtualHeight + vScrollBar1.LargeChange - 1;
```

Comment:
Seems like there's still a lot of code churn. Do you want me to accept the PR now? or give you more time to tweak?